hooks/environment

```diff
@@ -90,7 +90,8 @@ echo "--- :lock: Exchanging Token"
 dbg echo "-----Environment Variables-----"
 dbg printf "%s\n" "$(env | sort)"
 
-ci_token=$(buildkite-agent oidc request-token --audience ${BUILDKITE_PLUGIN_TOKEN_EXCHANGE_BUILDKITE_AUDIENCE})
+aud=${BUILDKITE_PLUGIN_TOKEN_EXCHANGE_BUILDKITE_AUDIENCE:-"identity-api"}
```

Comment:
Should these quotes be included in here?

Additionally, should we default the value to something more generic, such as "buildkite" or something?